src/redux/books/books.js

```diff
@@ -0,0 +1,24 @@
+const ADD_BOOK = 'add-book';
+const REMOVE_BOOK = 'remove-book';
+const defaultState = [];
+
+const BooksReducer = (state = defaultState, action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return {};
+    case REMOVE_BOOK:
+      return {};
+    default:
+      return state;
```

Comment:
- [ ] for the Add & Remove actions, currently you are returning an empty object, kindly make that each action does its roles, the ADD_BOOK should add a book and return a new state & same for the remove , kindly fix that

Reply:
I don't think that is a requirement for this project. Kindly read the requirements again.